inspire/modules/workflows/tasks/submission.py

```diff
@@ -287,3 +288,58 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def create_curation_ticket(template, queue="Test"):
```

Comment:
Yeah, we probably can pretty easily. I'll create a new commit to try it and then if it's ok I'll just squash it.